packages/ui/src/working-groups/types/ApplicationFormQuestionAnswer.ts

```diff
@@ -0,0 +1,13 @@
+import { ApplicationFormQuestionAnswerFieldsFragment } from '../queries'
+
+export interface ApplicationFormQuestionAnswer {
```

Comment:
WDYT about `QuestionAnswer`? I like descriptive types but maybe we will not have many question answers.

Or maybe `ApplicationQuestion` and `ApplicationQuestionAnswer` -> hence vomiting `Form` there.

Reply:
However, if you're fine with `ApplicationFormQuestionAnswer` you can leave it as is :)

Reply:
It's actually a scalding satire of java enterprise
I dropped the "Form", I feel like shortening it further either makes it too generic/non-descriptive or makes it linguistically unsound